### ISSUE TITLE
Rename settings for results window to mssql.results.[setting]

### ISSUE THREAD

**Issue by julielerman** [Microsoft/vscode-mssql#963]
- MSSQL Extension Version: 1.1.0
- VSCode Version: 1.14

The formatting settings all start with mssql.format. Maybe it would be nice to make the settings for results window be grouped like that as well?

**Comment by kevcunnane** [comment]
One of the things we looked at when doing this was best practices and also "what makes most sense" for us. Intellisense, Formatter and Query settings that start with `mssql.feature.xyz` all go to the backend service, while settings that are directly on `mssql.xyz` are front-end controlled. 

I'm also a little wary to change this as it might break existing users of the features. In the cases where we've changed it before now, it's generally been because the feature was broken (e.g. the setting didn't match the backend value) so there was no impact to users. Now that we're GA, I would prefer not to change unless necessary.

I'm leaning towards closing this as "Won't Fix", as I suspect due to the reasons above we'd prefer to keep these as-is. However when adding new settings, your suggestion makes sense and we can work to do a better job grouping them.
